mongoengine/connection.py

```diff
@@ -1,18 +1,20 @@
+from bson import string_type
```

Comment:
Its minor but import this from six (six.string_types) to keep it consistent with rest of the codebase

Reply:
makes sense

Reply:
fixed